src/common/components/play/GameBrowser.tsx

```diff
@@ -84,18 +79,18 @@ export default class GameBrowser extends React.Component<GameBrowserProps> {
     } else {
       return (
         <TableRow>
-          <TableRowColumn
-            colSpan={3}
+          <TableCell
+            colSpan={4}
```

Comment:
colspan changed?

Reply:
I think it was just wrong before, or it might have to do somehow with checkbox handling (that's no longer included in mui 1.x `Table`)